eclipse-theia/theia
# ovsx-client: update api version

#### What it does


Fixes: https://github.com/eclipse-theia/theia/issues/10221.

The commit updates the default **api-version** used when resolving extension-packs through our download script.
Since https://github.com/eclipse-theia/theia/pull/9959 bumped the default api, we had to also update the script defaults to properly respect the new default the framework supports.

#### How to test


- update the `package.json` to use the [builtin-extension-pack](https://open-vsx.org/api/eclipse-theia/builtin-extension-pack/1.50.1/file/eclipse-theia.builtin-extension-pack-1.50.1.vsi)
- delete the `plugins` folder if it exists (ex: `rm -rf plugins`)
- perform `yarn download:plugins`
- confirm that extensions are downloaded and resolved to their `1.53.2` version if available

#### Review checklist


#### Reminder for reviewers

- As a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)

Signed-off-by: vince-fugnitto <vincent.fugnitto@ericsson.com>
